### PR TITLE
Add Filestore monitoring dashboard integration

### PR DESCRIPTION
**Pull request by dadeeb**
Add Filestore monitoring dashboard integration to cloud monitoring.

### CODE REVIEW

File: dashboards/google-filestore/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+sample_dashboards:
+  -
+    category: Filestore
+    id: filestore
```
Comment (varun-c) [inline dashboards/google-filestore/metadata.yaml:4]: This should be the id of the sample_dashboard. In this case, that is `filestore_dashboard`.

File: dashboards/google-filestore/filestore_dashbard.json [hunk 2]
```diff
@@ -0,0 +1,444 @@
+{
```
Comment (varun-c) [inline dashboards/google-filestore/filestore_dashbard.json:1]: The filename has a typo that needs to be fixed for it to correctly show up in the UI.

**Review by varun-c — CHANGES_REQUESTED** [review]
The png file is not an image - please upload an image of how the dashboard is supposed to look once setup correctly.

File: dashboards/google-filestore/filestore_dashbard.json (hunk 2 above)
Comment (varun-c) [reply dashboards/google-filestore/filestore_dashbard.json:1]: The json should also exclude the "etag" and "name" fields.

**Comment by dadeeb** [comment]
Replaced the png file with a working png.
Removed "etag" & "name" from the json file.
Fixed metadata.yaml header to filestore_dashboard.

PTAL

File: dashboards/google-filestore/metadata.yaml (hunk 1 above)
Comment (varun-c) [reply dashboards/google-filestore/metadata.yaml:4]: My bad, this comment is confusing. On line 1, the field should still be `sample_dashboards`. On this line, the id should be the name of the sample dashboard being added (i.e. `filestore_dashboard`).

**Review by varun-c — APPROVED** [review]
Took another look and have one comment. The PR can be merged once that is resolved.

**Comment by dadeeb** [comment]
PTAL

**Comment by varun-c** [comment]
LGTM - merging it.

**Pull request merged**